finat/hdivcurl.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import absolute_import, division, print_function
+
+from six import iteritems
+
+from FIAT.reference_element import LINE
+
+import gem
+from finat.finiteelementbase import FiniteElementBase
+from finat.tensor_product import TensorProductElement
+
+
+class WrapperElementBase(FiniteElementBase):
+    """Common base class for H(div) and H(curl) element wrappers."""
+
+    def __init__(self, wrapped, transform):
+        super(WrapperElementBase, self).__init__()
+        self.wrapped = wrapped
+        self.transform = transform
```

Comment:
I would have written `wrappee` rather than `wrapped`, but meh.  Docstring for wrapped and transform though?